presto-docs/src/main/sphinx/admin/web-interface.rst

```diff
@@ -44,7 +79,10 @@ For more detailed information about a query, simply click the query ID link.
 The query detail page has a summary section, graphical representation of various stages of the
 query and a list of tasks. Each task ID can be clicked to get more information about that task.
 
-The summary section has a button to kill the currently running query. There are two visualizations
+The summary section has a button to kill the currently running query. Users can
+only kill queries, submitted with their username.
```

Comment:
Same as with view.  They can have permissions to kill other users queries.

Reply:
I removed that since its covered earlier.